person.rb

```diff
@@ -17,7 +17,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
   end
 
   def can_use_services?
-    true if of_age? || @parent_permission
+    of_age? || @parent_permission
   end
 
   def correct_name
```

Comment:
- [ ] Hi @akhror-valiev, you push fewer file changes hence I'm going to leave a general comment here.
> The methods in person.rb file still needs to refactor and get our code in compliance with the Single Responsibility Principle of SOLID.
 > - We can still create another class in a new file that handles specific tasks
 > - Each method in a class should perform a single responsibility. All requests for input and input validation should've been placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate input. Most of your methods are also carrying out validations and requesting input from the user which violates the SRP of SOLID. Let kindly take a look at the method below:

```ruby

def create_rental
    if @books.empty?
      puts 'Book list is empty please create a book first'
    else
      puts 'Select a book from the following list by number'
      books_list
      print '\nEnter book number: '
      book_id = gets.chomp.to_i

      puts 'Select a person from the following list by number (not id)'
      show_people
      print '\nEnter person number: '
      person_id = gets.chomp.to_i

      print '\nDate: '
      date = gets.chomp

      rent = Rental.new(date, @people[person_id], @books[book_id])
      @rental.push(rent)
      puts 'Rental created successfully!'
    end
```

 > - The method above is performing three duties
     > - create a rental
     > - validate inputs
     > - request for inputs

Let's kindly check through the other methods and refactor them once more so that our methods and classes can be maintainable in the future :+1: